libnetwork/drivers/bridge/bridge_linux_test.go

```diff
@@ -1396,6 +1345,16 @@ func TestCreateParallel(t *testing.T) {
 	}
 }
 
+func useStubFirewaller() func() {
+	origNewFirewaller := newFirewaller
+	newFirewaller = func(_ context.Context, config firewaller.Config) (firewaller.Firewaller, error) {
+		return firewaller.NewStubFirewaller(config), nil
+	}
+	return func() {
+		newFirewaller = origNewFirewaller
+	}
+}
```

Comment:
Use `t.Cleanup()` to save on boilerplate at the call sites?
```suggestion
func useStubFirewaller(t testing.TB) {
	origNewFirewaller := newFirewaller
	newFirewaller = func(_ context.Context, config firewaller.Config) (firewaller.Firewaller, error) {
		return firewaller.NewStubFirewaller(config), nil
	}
	t.Cleanup(func() {
		newFirewaller = origNewFirewaller
	})
}
```

Reply:
Done.

(The callers all also call `defer netnsutils.SetupTestOSContext(t)()` - I've not changed that, but I don't think there's any reason for them to be different.)